cv19ResSupportV3/V3/Infrastructure/HelpRequestsContext.cs

```diff
@@ -202,6 +204,22 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                             .HasColumnType("character varying");;
                     }
                 );
+                        modelBuilder.Entity<HelpRequestCallsEntity>(entity =>
+                            {
+                                entity.ToTable("help_request_calls");
+                                entity.HasKey(call => new {call.Id});
```

Comment:
Missed this before, apologies for late comment. Should add `.UseIdentityAlwaysColumn()` so that Id's would be autogenerated?

I'm not sure how the defaults work on Postgres EF Core, so I'd suggest to add that part explicitly just in case.

Since we're using Id as an identifier here, we're better of letting the DB autogenerate it, rather than implement that logic manually on the back-end,... or making the front-end populate it prior to doing POST.

Don't know what do you think?